src/domain/proposal.ts

```diff
@@ -204,6 +205,8 @@ export function updateCRProposal(
   setSchemeName(proposal.scheme, 'ContributionReward');
   getProposalIPFSData(proposal);
 
+  addRedeemableRewardOwner(proposalId, beneficiary);
```

Comment:
why is this needed ?

Reply:
To make sure the beneficiary is listed as a redeemer